public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx

```diff
@@ -99,18 +104,34 @@ export const LokiQueryBuilderOptions = React.memo<Props>(
               />
             </EditorField>
           )}
-          <EditorField
-            label="Resolution"
-            tooltip="Sets the step parameter of Loki metrics range queries. With a resolution of 1/1, each pixel corresponds to one data point. 1/10 retrieves one data point per 10 pixels. Lower resolutions perform better."
-          >
-            <Select
-              isSearchable={false}
-              onChange={onResolutionChange}
-              options={RESOLUTION_OPTIONS}
-              value={query.resolution || 1}
-              aria-label="Select resolution"
-            />
-          </EditorField>
+          {!isLogQuery && (
```

Comment:
Do we have tests that verify that these fields are hidden when it's a log query and shown with metric query?

Reply:
Added in https://github.com/grafana/grafana/pull/69648/commits/536437070d98a2aab972fa7aeeb4fde46c690374